vendor/bat-native-ledger/src/bat/ledger/internal/uphold/uphold_card.cc

```diff
@@ -382,6 +408,12 @@ void UpholdCard::OnCreateAnonAddressIfNecessary(
 void UpholdCard::CreateAnonAddress(
     ledger::ExternalWalletPtr wallet,
     CreateAnonAddressCallback callback) {
+  if (!wallet) {
```

Comment:
If `wallet` was a `nullptr` before this defensive code was added, could it have caused any side effects other than a crash?